### ISSUE TITLE
Error when opening tickets with cloned entities

### ISSUE THREAD

**Issue by verdanatech-marcio** [glpi-project/glpi#17009]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.15

### Bug description

When cloning an entity the sons_cache of the cloned entity are saved in the new one, when I try to create a new ticket with the new entity an error is displayed and it is not possible to create the ticket.
![image](https://github.com/glpi-project/glpi/assets/80538863/d4b9a12c-e908-4683-ba37-0dee44672417)

### Relevant log output

```shell
Resolved in this commit
https://github.com/verdanatech/glpi/commit/544afda12af8186b3a3079dc470b97eb6c2c0557
```

### Page URL

http://localhost/glpi/front/ticket.form.php

### Steps To reproduce

Clone an entity
Try opening a ticket with the cloned entity

### Your GLPI setup information

<details><summary>Instruções de instalação e configuração</summary><pre>GLPI 10.0.15 (/glpi-10.0.15 => /var/www/html/glpi-10.0.15)
Installation mode: TARBALL
Current language:pt_BR

</pre></details><details><summary>Server</summary><pre> 
Operating system: Linux vtcmtnb2590 6.1.0-20-amd64 #​1 SMP PREEMPT_DYNAMIC Debian 6.1.85-1 (2024-04-11) x86_64
PHP 7.4.33 apache2handler (Core, FFI, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, apcu, bcmath, bz2,
	calendar, ctype, curl, date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, igbinary, imap, intl, json, ldap,
	libxml, mbstring, mysqli, mysqlnd, openssl, pcre, pdo_mysql, posix, readline, redis, session, shmop, sockets, sodium, standard,
	sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="8M" safe_mode="" session.save_handler="files"
	upload_max_filesize="2M"
	disable_functions="pcntl_alarm,pcntl_fork,pcntl_waitpid,pcntl_wait,pcntl_wifexited,pcntl_wifstopped,pcntl_wifsignaled,pcntl_wifcontinued,pcntl_wexitstatus,pcntl_wtermsig,pcntl_wstopsig,pcntl_signal,pcntl_signal_get_handler,pcntl_signal_dispatch,pcntl_get_last_error,pcntl_strerror,pcntl_sigprocmask,pcntl_sigwaitinfo,pcntl_sigtimedwait,pcntl_exec,pcntl_getpriority,pcntl_setpriority,pcntl_async_signals,pcntl_unshare,"
	
Software: Apache ()
	Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
Server Software: Debian 12
	Server Version: 10.11.6-MariaDB-0+deb12u1
	Server SQL Mode: STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
	Parameters: verdanadesk@127.0.0.1/verdanadesk_glpi
	Host info: 127.0.0.1 via TCP/IP
	
PHP version (7.4.33) is supported.
Sessions configuration is OK.
Allocated memory is sufficient.
mysqli extension is installed.
Following extensions are installed: dom, fileinfo, filter, libxml, json, simplexml, xmlreader, xmlwriter.
curl extension is installed.
gd extension is installed.
intl extension is installed.
zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (10.11.6) is supported.
No files from previous GLPI version detected.
The log file has been created successfully.
Write access to /var/www/html/glpi-10.0.15/files/_cache has been validated.
Write access to /var/www/html/glpi-10.0.15/files/_cron has been validated.
Write access to /var/www/html/glpi-10.0.15/files has been validated.
Write access to /var/www/html/glpi-10.0.15/files/_dumps has been validated.
Write access to /var/www/html/glpi-10.0.15/files/_graphs has been validated.
Write access to /var/www/html/glpi-10.0.15/files/_lock has been validated.
Write access to /var/www/html/glpi-10.0.15/files/_pictures has been validated.
Write access to /var/www/html/glpi-10.0.15/files/_plugins has been validated.
Write access to /var/www/html/glpi-10.0.15/files/_rss has been validated.
Write access to /var/www/html/glpi-10.0.15/files/_sessions has been validated.
Write access to /var/www/html/glpi-10.0.15/files/_tmp has been validated.
Write access to /var/www/html/glpi-10.0.15/files/_uploads has been validated.

The following directories should be placed outside "/var/www/html/glpi-10.0.15":
‣ "/var/www/html/glpi-10.0.15/files" ("GLPI_VAR_DIR")
You can ignore this suggestion if your web server root directory is "/var/www/html/glpi-10.0.15/public".
PHP directive "session.cookie_httponly" should be set to "on" to prevent client-side script to access cookie values.
OS and PHP are relying on 64 bits integers.
exif extension is installed.
ldap extension is installed.
openssl extension is installed.
Following extensions are installed: bz2, Phar, zip.
Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to /var/www/html/glpi-10.0.15/marketplace has been validated.
Timezones seems loaded in database.

</pre></details><details><summary>GLPI constants</summary><pre> 
GLPI_ROOT: "/var/www/html/glpi-10.0.15"
GLPI_CONFIG_DIR: "/var/www/html/glpi-10.0.15/config"
GLPI_VAR_DIR: "/var/www/html/glpi-10.0.15/files"
GLPI_MARKETPLACE_DIR: "/var/www/html/glpi-10.0.15/marketplace"
GLPI_USE_CSRF_CHECK: "1"
GLPI_CSRF_EXPIRES: "7200"
GLPI_CSRF_MAX_TOKENS: "100"
GLPI_USE_IDOR_CHECK: "1"
GLPI_IDOR_EXPIRES: "7200"
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: false
GLPI_SERVERSIDE_URL_ALLOWLIST: ["/^(https?|feed):\\/\\/[^@:]+(\\/.*)?$/"]
GLPI_TELEMETRY_URI: "https://telemetry.glpi-project.org"
GLPI_INSTALL_MODE: "TARBALL"
GLPI_NETWORK_MAIL: "glpi@teclib.com"
GLPI_NETWORK_SERVICES: "https://services.glpi-network.com"
GLPI_MARKETPLACE_ALLOW_OVERRIDE: true
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: true
GLPI_USER_AGENT_EXTRA_COMMENTS: ""
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: "1"
GLPI_AJAX_DASHBOARD: "1"
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: "0"
GLPI_CENTRAL_WARNINGS: "1"
GLPI_TEXT_MAXSIZE: "4000"
GLPI_DOC_DIR: "/var/www/html/glpi-10.0.15/files"
GLPI_CACHE_DIR: "/var/www/html/glpi-10.0.15/files/_cache"
GLPI_CRON_DIR: "/var/www/html/glpi-10.0.15/files/_cron"
GLPI_DUMP_DIR: "/var/www/html/glpi-10.0.15/files/_dumps"
GLPI_GRAPH_DIR: "/var/www/html/glpi-10.0.15/files/_graphs"
GLPI_LOCAL_I18N_DIR: "/var/www/html/glpi-10.0.15/files/_locales"
GLPI_LOCK_DIR: "/var/www/html/glpi-10.0.15/files/_lock"
GLPI_LOG_DIR: "/var/www/html/glpi-10.0.15/files/_log"
GLPI_PICTURE_DIR: "/var/www/html/glpi-10.0.15/files/_pictures"
GLPI_PLUGIN_DOC_DIR: "/var/www/html/glpi-10.0.15/files/_plugins"
GLPI_RSS_DIR: "/var/www/html/glpi-10.0.15/files/_rss"
GLPI_SESSION_DIR: "/var/www/html/glpi-10.0.15/files/_sessions"
GLPI_TMP_DIR: "/var/www/html/glpi-10.0.15/files/_tmp"
GLPI_UPLOAD_DIR: "/var/www/html/glpi-10.0.15/files/_uploads"
GLPI_INVENTORY_DIR: "/var/www/html/glpi-10.0.15/files/_inventories"
GLPI_NETWORK_REGISTRATION_API_URL: "https://services.glpi-network.com/api/registration/"
GLPI_MARKETPLACE_PLUGINS_API_URI: "https://services.glpi-network.com/api/marketplace/"
GLPI_I18N_DIR: "/var/www/html/glpi-10.0.15/locales"
GLPI_VERSION: "10.0.15"
GLPI_SCHEMA_VERSION: "10.0.15@2eed74704cb07e0bac48b933cbd5c1c356f09629"
GLPI_MARKETPLACE_PRERELEASES: false
GLPI_MIN_PHP: "7.4.0"
GLPI_MAX_PHP: "8.4.0"
GLPI_YEAR: "2024"

</pre></details><details><summary>Libraries</summary><pre> 
htmlawed/htmlawed version 1.2.14 in (/var/www/html/glpi-10.0.15/vendor/htmlawed/htmlawed)
phpmailer/phpmailer version 6.8.0 in (/var/www/html/glpi-10.0.15/vendor/phpmailer/phpmailer/src)
simplepie/simplepie version 1.5.8 in (/var/www/html/glpi-10.0.15/vendor/simplepie/simplepie/library)
tecnickcom/tcpdf version 6.7.5 in (/var/www/html/glpi-10.0.15/vendor/tecnickcom/tcpdf)
michelf/php-markdown in (/var/www/html/glpi-10.0.15/vendor/michelf/php-markdown/Michelf)
true/punycode in (/var/www/html/glpi-10.0.15/vendor/true/punycode/src)
iamcal/lib_autolink in (/var/www/html/glpi-10.0.15/vendor/iamcal/lib_autolink)
sabre/dav in (/var/www/html/glpi-10.0.15/vendor/sabre/dav/lib/DAV)
sabre/http in (/var/www/html/glpi-10.0.15/vendor/sabre/http/lib)
sabre/uri in (/var/www/html/glpi-10.0.15/vendor/sabre/uri/lib)
sabre/vobject in (/var/www/html/glpi-10.0.15/vendor/sabre/vobject/lib)
laminas/laminas-i18n in (/var/www/html/glpi-10.0.15/vendor/laminas/laminas-i18n/src)
laminas/laminas-servicemanager in (/var/www/html/glpi-10.0.15/vendor/laminas/laminas-servicemanager/src)
monolog/monolog in (/var/www/html/glpi-10.0.15/vendor/monolog/monolog/src/Monolog)
sebastian/diff in (/var/www/html/glpi-10.0.15/vendor/sebastian/diff/src)
donatj/phpuseragentparser in (/var/www/html/glpi-10.0.15/vendor/donatj/phpuseragentparser/src/UserAgent)
elvanto/litemoji in (/var/www/html/glpi-10.0.15/vendor/elvanto/litemoji/src)
symfony/console in (/var/www/html/glpi-10.0.15/vendor/symfony/console)
scssphp/scssphp in (/var/www/html/glpi-10.0.15/vendor/scssphp/scssphp/src)
laminas/laminas-mail in (/var/www/html/glpi-10.0.15/vendor/laminas/laminas-mail/src/Protocol)
laminas/laminas-mime in (/var/www/html/glpi-10.0.15/vendor/laminas/laminas-mime/src)
rlanvin/php-rrule in (/var/www/html/glpi-10.0.15/vendor/rlanvin/php-rrule/src)
ramsey/uuid in (/var/www/html/glpi-10.0.15/vendor/ramsey/uuid/src)
psr/log in (/var/www/html/glpi-10.0.15/vendor/psr/log/Psr/Log)
psr/simple-cache in (/var/www/html/glpi-10.0.15/vendor/psr/simple-cache/src)
psr/cache in (/var/www/html/glpi-10.0.15/vendor/psr/cache/src)
league/csv in (/var/www/html/glpi-10.0.15/vendor/league/csv/src)
mexitek/phpcolors in (/var/www/html/glpi-10.0.15/vendor/mexitek/phpcolors/src/Mexitek/PHPColors)
guzzlehttp/guzzle in (/var/www/html/glpi-10.0.15/vendor/guzzlehttp/guzzle/src)
guzzlehttp/psr7 in (/var/www/html/glpi-10.0.15/vendor/guzzlehttp/psr7/src)
glpi-project/inventory_format in (/var/www/html/glpi-10.0.15/vendor/glpi-project/inventory_format/lib/php)
wapmorgan/unified-archive in (/var/www/html/glpi-10.0.15/vendor/wapmorgan/unified-archive/src)
paragonie/sodium_compat in (/var/www/html/glpi-10.0.15/vendor/paragonie/sodium_compat/src)
symfony/cache in (/var/www/html/glpi-10.0.15/vendor/symfony/cache)
html2text/html2text in (/var/www/html/glpi-10.0.15/vendor/html2text/html2text/src)
symfony/css-selector in (/var/www/html/glpi-10.0.15/vendor/symfony/css-selector)
symfony/dom-crawler in (/var/www/html/glpi-10.0.15/vendor/symfony/dom-crawler)
twig/twig in (/var/www/html/glpi-10.0.15/vendor/twig/twig/src)
twig/string-extra in (/var/www/html/glpi-10.0.15/vendor/twig/string-extra)
symfony/polyfill-ctype not found
symfony/polyfill-iconv not found
symfony/polyfill-mbstring not found
symfony/polyfill-php80 in (/var/www/html/glpi-10.0.15/vendor/symfony/polyfill-php80)
symfony/polyfill-php81 in (/var/www/html/glpi-10.0.15/vendor/symfony/polyfill-php81)
symfony/polyfill-php82 in (/var/www/html/glpi-10.0.15/vendor/symfony/polyfill-php82)
league/oauth2-client in (/var/www/html/glpi-10.0.15/vendor/league/oauth2-client/src/Provider)
league/oauth2-google in (/var/www/html/glpi-10.0.15/vendor/league/oauth2-google/src/Provider)
thenetworg/oauth2-azure in (/var/www/html/glpi-10.0.15/vendor/thenetworg/oauth2-azure/src/Provider)
phpCas version 1.6.0 in (/usr/share/php/CAS/source)

</pre></details><details><summary>SQL replicas</summary><pre> 
Not active

</pre></details><details><summary>Notifications</summary><pre> 
Way of sending emails: PHP

</pre></details><details><summary>Plugins list</summary><pre> 
	news                 Name: Alertas                        Version: 1.12.2     State: Not installed                           
		Install Method: Manual
	gantt                Name: gantt                          Version: 1.1.0      State: Not installed                           
		Install Method: Manual
	manufacturersimports Name: Importação de fornecedores     Version: 3.0.8      State: Enabled                                 
		Install Method: Manual
	skins                Name: Skins                          Version: 3.4.0      State: Enabled                                 
		Install Method: Manual
	chat                 Name: Verdana Chat                   Version: 12.3.1     State: Enabled                                 
		Install Method: Manual

</pre></details>

**Comment by cedric-anne** [comment]
@verdanatech-marcio 

Please open a pull request with your fix.

**Comment by verdanatech-marcio** [comment]
https://github.com/glpi-project/glpi/pull/17011

**Comment by cconard96** [comment]
Fixed by #17012
